### PR TITLE
[PyROOT experimental] Add __version__ to ROOT module

### PR DESCRIPTION
**Pull request by stwunsch**
I think it would be really nice to be compliant with other Python packages and provide a `__version__` magic for the ROOT module.

~~But be careful: This comes at the cost of a direct use of ROOT at startup time (non-lazy! Am I right?). I don't see how we could do this besides doing some template/macro magic and copying the version number to the sources at configure time.~~

Since the ROOT module is actually a class instance, we can add a `property`, which is evaluated lazily. Nice! :)

And added tests for the ROOT module!

### CODE REVIEW

**Comment by stwunsch** [comment]
Btw, `property` for modules does not exist. Afaik, this is the only lazy way to inject such a thing lazily.

**Comment by stwunsch** [comment]
Aaaaah wait, there is the ROOT facade! I'll update the PR :)

**Comment by stwunsch** [comment]
Fixed, there we go. Lazy version :)

**Comment by stwunsch** [comment]
@phsft-bot build with `-Dpyroot_experimental=ON`

**Comment by stwunsch** [comment]
Added tests for the ROOT module

File: bindings/pyroot_experimental/PyROOT/test/root_module.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    def test_version(self):
+        """
+        Test __version__ property
+        """
+        import ROOT
+        v = ROOT.__version__
+        self.assertTrue(type(v) == str)
+        self.assertIn("/", v)
+        self.assertIn(".", v)
```
Comment (etejedor) [inline bindings/pyroot_experimental/PyROOT/test/root_module.py]: This is great, thanks! Should we also compare here the version we obtain from `ROOT.__version__` to the one `gROOT.GetVersion` gives?

File: bindings/pyroot_experimental/PyROOT/test/root_module.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import unittest
+
+
+class ROOTModule(unittest.TestCase):
+    def test_import(self):
```
Comment (etejedor) [inline bindings/pyroot_experimental/PyROOT/test/root_module.py]: Can we have a docstring also at class level explaining what this class of tests does? Which is basically testing the functionality provided by the ROOT module facade.
E.g.
https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/PyROOT/test/rdataframe_makenumpy.py#L8

File: bindings/pyroot_experimental/PyROOT/test/root_module.py (hunk 1 above)
Comment (stwunsch) [reply bindings/pyroot_experimental/PyROOT/test/root_module.py]: Done!

File: bindings/pyroot_experimental/PyROOT/test/root_module.py (hunk 2 above)
Comment (stwunsch) [reply bindings/pyroot_experimental/PyROOT/test/root_module.py]: And done!

**Pull request merged**